scss/components/_messages.scss

```diff
@@ -17,27 +17,34 @@
 
 .message {
   @include clearfix();
-  border-top: 1px solid;
+  background-size: 2rem;
+  background-color: rgba($ivory-dark, .5);
+  border-color: $beige;
+  border-top: 2px solid $beige;
+  padding: 5rem 2rem 2rem 2rem;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `5rem 2rem 2rem` instead of `5rem 2rem 2rem 2rem`